### ISSUE TITLE
Make a digital/interactive version of the Union events timeline

### ISSUE THREAD

**Issue by bparslow** [thepoly/pipeline#109]
We published [this timeline of Union-related events](https://github.com/thepoly/pipeline/files/3628597/Timeline.of.Union.events.pdf) in an NRB issue. I think it's a valuable resource for the community and would be a great (more permanent) addition to our site.

**Comment by pantp** [comment]
Timeline style [option 1](https://bootsnipp.com/snippets/xrKXW) or [option 2](https://bootsnipp.com/snippets/OzQj). I really like option 1. @bparslow

**Comment by pantp** [comment]
[Here](https://bootsnipp.com/snippets/Q0ppE) are some more options.

**Comment by bparslow** [comment]
Could we replace the graphics with our pics in the most recent one?

**Comment by pantp** [comment]
Maybe? I do think the last link I sent you is too flashy compared to the rest of the Poly site.

**Comment by bparslow** [comment]
I totally agree, but I like the idea of that image placement a lot so that the timeline doesn't get too too long vertically

**Comment by pantp** [comment]
@bparslow I attempted to move this page to the News section in my latest commit. There is still some work that needs to be done with this. If you want to look into it, basically we need to display the Union Timeline in article_index_page.html somehow. This means Union Timeline needs to be inserted into the list of articles and sorted by date.
